templates/firefox-installer/install/install.1.parquetmr.txt

```diff
@@ -52,4 +52,5 @@ message install {
   optional binary attribution (UTF8);
   optional binary profile_cleanup_prompt (UTF8);
   optional boolean profile_cleanup_requested;
+  optional boolean silent;
```

Comment:
The parquet schemas have fallen out of favor, only updates to the JSON Schemas will show up in changes to the BigQuery tables.

Reply:
Can the readme be updated to reflect that? As someone who only interacts with these things occasionally, it's very difficult to keep up with everything that's always changing, and a little bit of updated documentation would go a long way.